src/lib/components/SelectField/README.md

```diff
@@ -0,0 +1,681 @@
+        required
+        validationState="invalid"
+        validationText="You must select at least one kind of fruit."
+        value={fruit}
+      />
+      <SelectField
+        label="Your favourite fruit"
+        onChange={(e) => setFruit(e.target.value)}
+        options={options}
+        required
+        validationState="valid"
+        validationText="Great, they're in stock!"
+        value={fruit}
+        variant="filled"
+      />
+      <SelectField
+        label="Your favourite fruit"
+        onChange={(e) => setFruit(e.target.value)}
+        options={options}
+        required
+        validationState="warning"
+        validationText="Oh, really?"
+        value={fruit}
+        variant="filled"
+      />
+      <SelectField
+        label="Your favourite fruit"
+        onChange={(e) => setFruit(e.target.value)}
+        options={options}
+        required
+        value={fruit}
+        validationState="invalid"
+        validationText="You must select at least one kind of fruit."
+        variant="filled"
+      />
+    </React.Fragment>
+  );
+})
+```
+
+### Disabled State
+
+It's possible to disable just some options or the whole input.
+
+```docoff-react-preview
+React.createElement(() => {
+  const [fruit, setFruit] = React.useState('apple');
+  const options = [
+    {
+      label: 'Apple',
+      value: 'apple',
+    },
+    {
+      disabled: true,
+      label: 'Banana',
+      value: 'banana',
+    },
+    {
+      label: 'Grapefruit',
+      value: 'grapefruit',
+    },
+  ];
+  return (
+    <React.Fragment>
+      <SelectField
+        label="Your favourite fruit"
+        onChange={(e) => setFruit(e.target.value)}
+        options={options}
+        value={fruit}
+      />
+      <SelectField
+        label="Your favourite fruit"
+        onChange={(e) => setFruit(e.target.value)}
+        options={options}
+        value={fruit}
+        variant="filled"
+      />
+      <SelectField
+        disabled
+        label="Your favourite fruit"
+        onChange={(e) => setFruit(e.target.value)}
+        options={options}
+        value="apple"
+      />
+      <SelectField
+        disabled
+        label="Your favourite fruit"
+        onChange={(e) => setFruit(e.target.value)}
+        options={options}
+        value="apple"
+        variant="filled"
+      />
+    </React.Fragment>
+  );
+})
+```
+
+## Forwarding HTML Attributes
+
+In addition to the options below in the [component's API](#api) section, you
+can specify [React synthetic events] or you can **add whatever HTML attribute
+you like.** All attributes that don't interfere with the API are forwarded to
+the native HTML `<select>`. This enables making the component interactive and helps
+to improve its accessibility.
+
+👉 Refer to the MDN reference for the full list of supported attributes of the
+[select] element.
+
+## Forwarding ref
+
+If you provide [ref], it is forwarded to the native HTML `<select>` element.
+
+## API
```

Comment:
it does not fit to the screen.

![image](https://user-images.githubusercontent.com/10213870/209101737-912f0c69-baea-4f04-bd3f-395bebd64f39.png)

Reply:
Agreed with @davidurban that he will take a look at it.

Reply:
Should be fixed now.

Reply:
![image](https://user-images.githubusercontent.com/10213870/212316653-798a5d9c-1ab5-477a-868c-e68bd99e06a0.png)
It does not work on narrow screens.

Reply:
![image](https://user-images.githubusercontent.com/10213870/212316835-e68da58b-03a5-4e5c-a4bf-9bce589ee1c8.png)
And it does not work on medium screen either. This is not usable. If you need help, @adamkudrna will probably now simple and correct answer to this problem.